SwiftGit2Tests/RepositorySpec.swift

```diff
@@ -641,8 +641,65 @@ class RepositorySpec: QuickSpec {
 				expect(commitMessages).to(equal(expectedMessages))
 			}
 		}
+
+		describe("Repository.status	") {
+			it("Should return accurate status") {
+				// Repo with no status
+				let repo = Fixtures.mantleRepository
+				let branch = repo.localBranch(named: "master").value!
+				expect(repo.checkout(branch, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let status = repo.status()
+
+				expect(status.value?.count).to(equal(0))
+
+				let repoWithStatus = Fixtures.sharedInstance.repository(named: "repository-with-status")
+				let branchWithStatus = repoWithStatus.localBranch(named: "master").value!
+				expect(repoWithStatus.checkout(branchWithStatus, strategy: CheckoutStrategy.None)).to(haveSucceeded())
+
+				let statusWithStatus = repoWithStatus.status()
+
+				expect(statusWithStatus.value?.count).to(equal(5))
```

Comment:
This should verify that the actual status are correct—not just verify the total number of statuses. Or there should be separate tests that verify individual statuses.